README.md

```diff
@@ -104,6 +105,82 @@ $ remme account get-balance \
 368440.0
 ```
 
+### Batches
+
+Get batch by its identifier — ``remme batch get-single``:
+
+| Arguments | Type   |  Required | Description                                         |
+| :-------: | :----: | :-------: | --------------------------------------------------- |
+| id        | String |  Yes      | Identifier to get a batch by.                       |
+| node-url  | String |  No       | Node URL to apply the command to.                   |
+
+```bash
+$ remme batch get-single \
+      --id=4da4185d8c69ade0d1e85a8c260554e8d14f072af9f0629a14b840c52abc92142617024977765abd240735b7bca4b1c44ce1034e964e11083c7405c7adcc6448 \
+      --node-url=node-genesis-testnet.remme.io
+{
+    "data": {
+        "header": {
+            "signer_public_key": "03d425d2d17b64e3ef8fee028089a567fbb05bd556f98c0b6fb62bc5750ea62b8f",
+            "transaction_ids": [
+                "03a7cbd52382dc2b3f542fc208e2b4bdb83d57e31559cba943b598839bcdf72f161d0ee624c8cc1ea93ce712affa94ef3001b5c2af47499a96a24de180b4c19e",
+                "bdefc656a479b201b87b87d4648bed04f908da236d89317c6e4f2cc10f1db25366d332093bad29c5ca0cde0a37af4a0ebb522c8092b6278c73f93db2dacb2035"
+            ]
+        },
+        "header_signature": "4da4185d8c69ade0d1e85a8c260554e8d14f072af9f0629a14b840c52abc92142617024977765abd240735b7bca4b1c44ce1034e964e11083c7405c7adcc6448",
+        "trace": false,
+        "transactions": [
```

Comment:
Remove the last transaction to make the size of the example less using triple dots to say it is still a list.

```
{
    'transactions': [
        {
           <only one transaction>
        },
        ...   <- say it is a list of transactions
    ]
}
```

Reply:
Done.